hand.py

```diff
@@ -0,0 +1,41 @@
+"""a class for storing poker hands"""
+
+from deck import Deck
+from card import Card
+
+class Hand:
+    def __init__(self, hand_list):
+        self.hand_list = hand_list
+
+    def __eq__(self, other):
+        """Return eq
+        >>> (
+        ... Hand([Card('H', '5'), Card('C', '2')]) ==
+        ... Hand([Card('H', '5'), Card('C', '2')])
+        ... )
+        True
+        >>> (
+        ... Hand([Card('H', '5'), Card('C', '3')]) ==
+        ... Hand([Card('H', '5'), Card('C', '2')])
+        ... )
+        False
+        """
+        return(
+            self.hand_list == other.hand_list
+        )
+
+    def __repr__(self):
+        """Return repr.
+        >>> repr(Hand([Card('H', '5'), Card('C', '2')]))
+        "Hand([Card('H', '5'), Card('C', '2')])"
+        """
+        return 'Hand({!r})'.format(
+            self.hand_list
+        )
+
+def generate_hand(deck):
+    """adds a card to a hand
```

Comment:
Couldn't this just be an outcome of the first two passes through the deal loop?  The hand can initialize to be empty (i.e., no cards) then.